tests/specs/SchemaFuzzSpec.elm

```diff
@@ -31,14 +31,22 @@ spec =
             , array [ items (string []), minItems 5 ]
             , array [ items (string []), maxItems 10 ]
             , array [ items (string []), minItems 5, maxItems 10 ]
+            , object
```

Comment:
It's working because the `JsonSchema.Validator` module is not yet checking these rules.